DuckDuckGo/MainWindow/MainWindowController.swift

```diff
@@ -74,8 +74,12 @@ final class MainWindowController: NSWindowController {
         return false
 #elseif REVIEW
         if Application.runType == .uiTests {
+            Application.appDelegate.onboardingStateMachine.state = .onboardingCompleted
             return false
         } else {
+            if Application.runType == .uiTestsOnboarding {
```

Comment:
❓ If this line was added, is the line on 83 needed? It seems that we set the onboarding to completed, but then we check again in the view model.

Reply:
In reality that variable is a bit unclear but since in the Onboarding ui test we are not testing the contextual onboarding this avoids it to appear after the linear onboarding test completes (which is when the contextual onboarding would start and would affect tests run afterwards)